pkg/cmd/subcommand/install.go

```diff
@@ -203,26 +212,41 @@ func (i *Install) RunInstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 		}
 	}
 
+	// Record the list of components
+	// that currently exist in the cluster.
+	if _, err := operator.GetInventoryRecord(ctx, false); err != nil {
+		return errors.Wrap(err, "failed to get inventory record")
+	}
+	defer operator.RecordInventory(ctx)
+
+	c := make(chan os.Signal)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-c
+		operator.RecordInventory(ctx)
```

Comment:
what if calling done() here to inform all goroutines to cancel and exit from the main instead of calling below codes in a goroutine?
```
    operator.RecordInventory(ctx)
    os.Exit(1)
```

Reply:
done() is better